app/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/returns/triage/CommonTriageQuestionsController.scala

```diff
@@ -516,7 +519,8 @@ class CommonTriageQuestionsController @Inject() (
               draftReturn = DraftMultipleDisposalsReturn.newDraftReturn(
                 fillingOutReturn.draftReturn.id,
                 newTriageAnswers,
-                fillingOutReturn.draftReturn.representeeAnswers
+                if (furtherReturn) Some(RepresenteeAnswers.IncompleteRepresenteeAnswers.empty)
```

Comment:
this shouldn't be unset for a further return - it should just be what was there originally